open-metadata-implementation/adapters/open-connectors/repository-services-connectors/open-metadata-collection-store-connectors/docs/README.md

```diff
@@ -26,4 +26,77 @@ metadata in an existing metadata repository.
 * **[Overview of the Event Mapper Connector Interface](overview-of-the-event-mapper-connector-interface.md)** -
 a walk-through of the methods on the event mapper connector interface.  This connector enables
 events from an existing metadata repository that indicate that metadata is changing, to be passed
-to the OMRS so it can be distributed to the other metadata repositories who are members of the same cohort.
\ No newline at end of file
+to the OMRS so it can be distributed to the other metadata repositories who are members of the same cohort.
+
+## Getting Started Implementing a New Connector
+
+As implied by the overview above, a logical place to start in implementing a new
+Open Metadata Collection Store Connector is by implementing the OMRS interface.
+
+This requires implementing:
+
+- an `OMRSRepositoryConnectorProvider` specific to your connector, which extends `OMRSRepositoryConnectorProviderBase`.
+    This really only involves defining the connector class and setting this in the constructor.
+- an `OMRSRepositoryConnector` specific to your connector, which extends `OMRSRepositoryConnector`. Most likely the
+    main logic of this class will be implemented by overriding the `initialize()` method of the base class,
+    and putting into this any logic for initializing the connection: for example, connecting to an underlying
+    database, or starting a REST API session, etc. (Similarly, override the `disconnect()` method to properly
+    cleanup / close such resources.)
+- an OMRSMetadataCollection specific to your repository, which extends `OMRSMetadataCollectionBase`. Ideally your
+    implementation should override each of the methods defined in the base class, but a logical place to start is
+    likely to be the `getEntityDetail()` method that retrieves an entity by its GUID.
+
+Once these minimal starting points are implemented, you should be able to configure the
+[OMAG server chassis](../../../../../governance-servers/server-chassis/server-chassis-spring/README.md)
+as a proxy to your repository connector by following the instructions in
+[using the admin services](../../../../../governance-servers/admin-services/Using-the-Admin-Services.md)
+to configure it as a proxy (and providing the name of your `OMRSRepositoryConnectorProvider` class).
+
```

Comment:
Typically this only works if you put your connector into the adapter module.  If built outside of Egeria then need to create your own aserver chassis - as was done with the original igc repository proxy.

Reply:
Hmmm... maybe I need to re-think the "quickstart" idea here then, along with the point above on patterns. Is there a simple way for a developer to quickly standup a basic scaffold (not end-state) without needing to build their own server, update all the admin classes, etc?